### ISSUE TITLE
能否加入自定义system prompt功能

### ISSUE THREAD

**Issue by minecraft8093** [Chanzhaoyu/chatgpt-web#329]
可以有效的设置前置条件

**Comment by Yardanico** [comment]
Yes, ability to customize the system prompt would be very nice, especially if it could be done per-chat. System prompts with ChatGPT API allow to change the AI's behaviour in a very wide way - e.g. you can even make it roleplay a character.

**Comment by lingyu123-su** [comment]
期待+1

**Comment by mm519897405** [comment]
我也想问，不知道怎么搞

**Comment by tastypear** [comment]
现在会发送示例作为system prompt，也就是下面这段：You are ChatGPT, a large language model trained by OpenAI. Answer as concisely as possible. Knowledge cutoff: 2021-09-01……

我的临时解决办法是，建一个接口转发，让chatgpt先post到我这里，把system内容替换掉再传给服务器。我的prompt带一个特殊的短语，供接口转发时处理system中的内容。

但是这个方法很麻烦，每个会话都能自定义system prompt才是比较好的方式。
